test/puppetlabs/puppetdb/http/aggregate_event_counts_test.clj

```diff
@@ -91,7 +91,48 @@
          ["=" "certname" "foo.local"]
          ["<=" "report_receive_time" current-time-str]
          ["<=" "run_start_time" current-time-str]
-         ["<=" "run_end_time" current-time-str])))
+         ["<=" "run_end_time" current-time-str]))
+
+  (store-example-report! (:basic2 reports) (coerce/to-string (now)))
+
+  (are [result query] (= result
+                         (query-result method endpoint query
+                                       {:summarize_by "resource"
+                                        :distinct_resources true
+                                        :distinct_start_time 0
+                                        :distinct_end_time (now)}))
```

Comment:
Do `:distinct_start_time` and `:distinct_end_time` parameters need to be provided even if `["=" "latest_report?" true]` query is used?